src/pages/FeelGoodPage.js

```diff
@@ -0,0 +1,55 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import pinkArrow from '../Images/arrow_pink.svg';
+
+export default function FeelGoodPage() {
+  const navigate = useNavigate();
+  return (
+    <Flex>
+      <GoBackButton onClick={() => navigate(-1)}>
+        <img
+          alt="Red Arrow, if clicked leads back to main page"
+          src={pinkArrow}
+          width="60"
+        />
+      </GoBackButton>
```

Comment:
Could you use the Button wrapper for something else than the return functionality (if you would give it another img for example)? If yes, you should choose a more general name like 'ButtonWithImage' or 'CompactButton'. Then you can reuse this component whereever you need a button with an icon.